daml-foundations/daml-tools/da-hs-daml-cli/DA/Cli/Damlc.hs

```diff
@@ -348,7 +350,9 @@ execBuild options mbOutFile initPkgDb projectCheck = do
                     pVersion
                     pExposedModules
                     pDependencies
-        Managed.with (Compiler.newIdeState opts Nothing loggerH) $ \compilerH -> do
+        let eventLogger (EventFileDiagnostics (fp, diags)) = printDiagnostics $ map (fp,) diags
```

Comment:
Have you checked what happens if we get an error instead of a warning? My understanding is that those are already displayed without your PR so I would expect that they are now displayed twice?

Reply:
You're right, it now displays errors twice! Time to fix that.